pkg/webui/components/code-editor/code-editor.styl

```diff
@@ -14,7 +14,7 @@
 
 .wrapper
   border-input()
-  border-radius: $br.s
+  border-radius: $br.xs
```

Comment:
Not sure if this is also affected by the resolution issue, but if not then we might rather change the variable value than changing this in every stylesheet.